module_config/module_argon/sdkconfig.defaults

```diff
@@ -266,10 +266,14 @@ CONFIG_NEWLIB_STDIN_LINE_ENDING_CRLF=
 CONFIG_NEWLIB_STDIN_LINE_ENDING_LF=
 CONFIG_NEWLIB_STDIN_LINE_ENDING_CR=y
 CONFIG_NEWLIB_NANO_FORMAT=y
-CONFIG_CONSOLE_UART_DEFAULT=y
-CONFIG_CONSOLE_UART_CUSTOM=
+CONFIG_CONSOLE_UART_DEFAULT=
+CONFIG_CONSOLE_UART_CUSTOM=y
 CONFIG_CONSOLE_UART_NONE=
-CONFIG_CONSOLE_UART_NUM=0
+CONFIG_CONSOLE_UART_CUSTOM_NUM_0=
+CONFIG_CONSOLE_UART_CUSTOM_NUM_1=y
+CONFIG_CONSOLE_UART_NUM=1
+CONFIG_CONSOLE_UART_TX_GPIO=17
```

Comment:
This is a secondary UART, right? I don't think enabling this is going to be useful, as it's not possible to easily get access to these pins, so potentially leave disabled as previously?

Reply:
@avtolstoy Yeah, it's the secondary UART. We previously used the first UART (UART0) for logging, this is not correct because UART0 is used for AT command. The reason we need the log is by the Realtek team, they help burn the MCU and need us to output the firmware version log to confirm the firmware burnt is correct. We also have the test points on Argon/Tracker to capture the error log if necessary.